src/main/java/com/paskar/email/application/service/EmailRepositoryForHibernateIml.java

```diff
@@ -0,0 +1,51 @@
+package com.paskar.email.application.service;
+
+import com.paskar.email.application.model.Email;
+import com.paskar.email.application.repositiory.hibernate.CustomRepositoryForProject;
+import com.paskar.email.application.repositiory.hibernate.EmailRepoForHibernate;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+@RequiredArgsConstructor
+@Transactional
```

Comment:
This is fine, but you have read only operations.
Read operations should be in read only transaction!